Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -187,6 +187,13 @@
     return [m_pdfDocument isLocked];
 }
 
+void PDFPluginBase::notifySelectionChanged()
+{
+    if (!m_frame || !m_frame->page())
+        return;
+    m_frame->protectedPage()->didChangeSelection(*m_frame->protectedCoreLocalFrame());
```

Comment:
We should think a little bit about whether it makes sense to trigger `didChangeSelection()` for inside-PDF selections, Probably fine, but other code might get confused about selection in HTML content not actually changing.

Reply:
Probably fine and also not doing much? Maybe hortont424 knows more